server/src/endpoints/qs/mod.rs

```diff
@@ -6,12 +6,17 @@ use actix_web::{
     web::{self, Data},
     HttpResponse, Responder,
 };
+use itertools::Itertools;
```

Comment:
Can we avoid adding `itertools` as dependency? Using `.format(...)` is convenient, but can't we get the same effect using `map` and `join`? It's probably more allocations, but it's only in case of an error and it's not going to be that many versions.

Reply:
Sure. 

I started to replace it and realized that `join` is only available for a slice of strings, however here we have a slice of `ApiVersion`'s. I would need to collect them into vec. WDYT?

Reply:
Since this only happens in case of an error, I'm not too worried about the additional allocations if that's what you mean. Personally I'd rather avoid the additional dependency.

Reply:
Removed Itertools dependency: 98c47d3